### PR TITLE
correct sshMaster0 port number

### PR DESCRIPTION
**Pull request by rjtsdl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] The PR has modified HISTORY.rst with an appropriate description of the change (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

### Command Guidelines

- [x] Each command and parameter has a meaningful description.
- [x] Each new command has a test.

(see [Authoring Command Modules](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules))

### CODE REVIEW

**Comment by rjtsdl** [comment]
Fix #4210

**Comment by rjtsdl** [comment]
@derekbekoe , This is already the 5th pr i made on az cli repo. I would like to get the write access to this repo, so I won't need to ping someone every time, to help me merge. Is this a legit request? Do you know any process i need to go through? Thx

File: src/command_modules/azure-cli-acs/HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+unreleased
++++++++++++++++++++
+* minor fix
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-acs/HISTORY.rst]: "Correct ssh master port number from 2200 to 22"

**Comment by derekbekoe** [comment]
@rjtsdl We require at least one review from the team before merging PRs and we monitor for incoming PRs. This PR was only created 15 mins ago but I've reviewed it just now. Thanks.

**Comment by rjtsdl** [comment]
@derekbekoe , i understand the policy to have at least one approver from the team. I just want to get the privilege of merge, so when i get approval, i can merge it myself.

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 2]
```diff
@@ -682,14 +682,19 @@ def _create(resource_group_name, deployment_name, dns_name_prefix, name, ssh_key
...
-            "value": "[concat('ssh ', '{0}', '@', reference(concat('Microsoft.ContainerService/containerServices/', '{1}')).masterProfile.fqdn, ' -A -p 2200')]".format(admin_username, name)  # pylint: disable=line-too-long
+            "value": "[concat('ssh ', '{0}', '@', reference(concat('Microsoft.ContainerService/containerServices/', '{1}')).masterProfile.fqdn, ' -A -p 22')]".format(admin_username, name)  # pylint: disable=line-too-long
         },
     }
     if orchestrator_type.lower() != "kubernetes":
         outputs["agentFQDN"] = {
             "type": "string",
             "value": "[reference(concat('Microsoft.ContainerService/containerServices/', '{}')).agentPoolProfiles[0].fqdn]".format(name)  # pylint: disable=line-too-long
         }
+        # override sshMaster0 for non-kubernetes scenarioes
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: typo with 'scenarioes'

**Comment by derekbekoe** [comment]
@rjtsdl only the CLI team has merge permissions so we can coordinate PR merges.

**Pull request merged**